Geometry/HGCalGeometry/src/HGCalGeometryLoader.cc

```diff
@@ -114,6 +114,8 @@ HGCalGeometry* HGCalGeometryLoader::build (const HGCalTopology& topology) {
       }
     }
   }
+  // FIXME: Sort valid DetIds if neded
+  // geom->sortDetIds();
```

Comment:
@ianna better to just go ahead and sort them. It shouldn't have any adverse effect and can only help in the future.